paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/SupportedPaymentMethodKtx.kt

```diff
@@ -0,0 +1,208 @@
+package com.stripe.android.paymentsheet.model
```

Comment:
These functions all came from the SupportedPaymentMethod file.   Some functions have an added LpmRepository function, and some are extensions from the SupportedPaymentMethod data classes because they need access to the fields in SupportedPaymentMethod, but need to be in this module to access the PaymentSheet.Configuration.

Hopefully most of this logic will eventually move to a server.